src/west/main.py

```diff
@@ -390,6 +390,8 @@ def set_zephyr_base(args):
         zb_prefer = config.config.get('zephyr', 'base-prefer',
                                       fallback=None)
         zb_config = config.config.get('zephyr', 'base', fallback=None)
+        if not os.path.isabs(zb_config):
```

Comment:
What is `os.path.isabs(None)` in case there's no config value for it?

Reply:
good catch, fixed.